client/stylesheets/_modal.scss

```diff
@@ -371,6 +371,16 @@
 	margin: 8px 0 15px 0 !important;
 }
 
+.profileReceivedPriceOriginal {
+	font-size: 0.8rem;
+	vertical-align: middle;
```

Comment:
Line should be indented with spaces, not tabs<br>Properties should be ordered font-size, margin-left, vertical-align